official/vision/beta/projects/yolo/configs/backbones.py

```diff
@@ -12,26 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Lint as: python3
-
 """Backbones configurations."""
-
 import dataclasses
-
 from official.modeling import hyperparams
-
 from official.vision.beta.configs import backbones
 
 
 @dataclasses.dataclass
 class Darknet(hyperparams.Config):
-  """Darknet config."""
-  model_id: str = 'darknet53'
-  width_scale: float = 1.0
-  depth_scale: float = 1.0
+  """DarkNet config."""
+  model_id: str = 'cspdarknet53'
+  width_scale: int = 1.0
```

Comment:
1.0 is not an int, can you use 1 here and line below?

Reply:
changed to float